.eslintrc.json

```diff
@@ -1,4 +1,20 @@
 {
+  "root": true,
   "extends": "hexo",
-  "root": true
-}
\ No newline at end of file
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaVersion": 2020
+  },
+  "env": {
+    "es6": true,
+    "node": true,
```

Comment:
We defined `es6`, `node` in https://github.com/hexojs/eslint-config-hexo/blob/de333162ba947a9bf819f98bb9394b0b49041e26/eslint.js#L141-L142.

So, no need these lines.